Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
```

Comment:
Concerns over the existence of `USBCON` were resolved with https://github.com/MarlinFirmware/Marlin/pull/6737#issuecomment-302045819